test/scenario-test.ts

```diff
@@ -6,6 +6,7 @@ import * as path from 'path';
 import { CodeGenConstants, CompatibleLevel, GenerateSdk, TargetMode } from '../src/utils/models';
 import { copyRecursiveSync, deleteFolderRecursive, isNullOrUndefined } from '../src/utils/helper';
 import * as sourceMapSupport from 'source-map-support';
+import { Test } from 'mocha';
```

Comment:
I think we no longer use mocha for testing.

Reply:
Jest pr hasn't been merged yet. This is for purely bugfixing